bluemira/optimisation/_nlopt/conditions.py

```diff
@@ -51,6 +56,14 @@ def _validate(self) -> None:
             self.max_eval = int(self.max_eval)
 
     def _no_stopping_condition_set(self) -> bool:
+        """
+        Checks if all stopping conditions are unset.
+
+        Returns
+        -------
+        bool :
```

Comment:
Please remove the return types from the docstrings they are in the function definition. You just need the ':' as commented in the original issue

Reply:
done